python/paddle/fluid/optimizer.py

```diff
@@ -540,6 +549,228 @@ def _append_optimize_op(self, block, param_and_grad):
         return momentum_op
 
 
+class DGCOptimizer(MomentumOptimizer):
```

Comment:
should be called `DGCMomentumOptimizer`

Or, a better way is like: 

```
optimizer = fluid.optimizer.Momentum()
optimizer = fluid.distributed.DGCOptimize(optimizer)
```

Reply:
Done.Thanks!